core/transport/src/main/proto/grpc/raft_journal.proto

```diff
@@ -0,0 +1,73 @@
+syntax = "proto2";
+
+option java_multiple_files = true;
+option java_package = "alluxio.grpc";
+option java_outer_classname = "RaftJournalProto";
+
+package alluxio.grpc.meta;
+
+import "grpc/common.proto";
+
+message JournalQueryRequest {
+  optional GetSnapshotInfoRequest snapshotInfoRequest = 1;
+  optional GetSnapshotRequest snapshotRequest = 2;
+}
+
+message JournalQueryResponse {
+  optional GetSnapshotInfoResponse snapshotInfoResponse = 1;
+}
+
+message GetSnapshotInfoRequest {
+}
+
+message GetSnapshotInfoResponse {
```

Comment:
Where is this used in the proto files? I can't seem to find it.

Reply:
It is not used in proto file. It is used in Ratis messaging system in `SnapshotReplicationManager::handleRequest`